lib/Bio/EnsEMBL/DataCheck/Checks/CompareProjectedSynonyms.pm

```diff
@@ -0,0 +1,80 @@
+=head1 LICENSE
+
+Copyright [2018-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CompareProjectedSynonyms;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'CompareProjectedSynonyms',
+  DESCRIPTION    => 'Compare Projected Synonyms counts between two databases, categorised by db_name coming from the external_db',
+  GROUPS         => ['compare_core', 'xref'],
+  DATACHECK_TYPE => 'advisory',
+  DB_TYPES       => ['core'],
+  TABLES         => ['xref','external_db','external_synonym','object_xref']
+};
+
+sub tests {
+  my ($self) = @_;
+
+  SKIP: {
+    my $old_dba = $self->get_old_dba();
+
+    skip 'No old version of database', 1 unless defined $old_dba;
+
+    $self->projected_synonyms_counts($old_dba);
+  }
+}
+
+sub projected_synonyms_counts {
+  my ($self, $old_dba) = @_;
+
+  my $threshold = 0.78;
```

Comment:
This threshold seems arbitrarily specific. Might be worth looking across all the 'Compare' tests and settle on something consistent - the CompareSynonym and CompareGOXref use 0.70, CompareProjectedGOXrefs has 0.90. Maybe 0.80?

Reply:
Sure, that make perfect sense. I have updated the other Compare* to 0.80.